lib/raven/client.rb

```diff
@@ -104,11 +104,7 @@ def generate_auth_header
     end
 
     def strict_encode64(string)
-      if Base64.respond_to? :strict_encode64
-        Base64.strict_encode64 string
-      else # Ruby 1.8
-        Base64.encode64(string)[0..-2]
-      end
+      Base64.strict_encode64(string) if Base64.respond_to?(:strict_encode64)
```

Comment:
Do you need to keep the `respond_to?` check around? Seems like that's just there for 1.8 compatibility